airo-camera-toolkit/airo_camera_toolkit/cameras/multiprocess/multiprocess_rgbd_camera.py

```diff
@@ -138,6 +138,10 @@ def _retrieve_depth_image(self) -> NumpyIntImageType:
     def _retrieve_colored_point_cloud(self) -> PointCloud:
         positions = self._last_frame.point_cloud[0]
         colors = self._last_frame.point_cloud[1]
+        # Remove NaN values from the point cloud, if any. This can happen if the camera returns a sparse point cloud, like Realsense.
+        mask = np.isfinite(positions).all(axis=1)
```

Comment:
@m-decoster I think a similar check caused a performance penalty in the past. Maybe a good idea to measure how long this takes?